workshop/exercises/docker-compose.yml

```diff
@@ -44,4 +44,4 @@ services:
 
     volumes:
       - ./pygeoapi.config.yml:/pygeoapi/local.config.yml
-      # - ./data:/data # Exercise 1 - First - Ready to pull data from here
+      - ./data:/data # Exercise 1 - First - Ready to pull data from here
```

Comment:
We should leave this commented out. Students will uncomment it, once they arrive to this exercise. Just remember to explain that in the text:  https://dive.pygeoapi.io/publishing/ogcapi-maps/